test/ctia/bulk/routes_test.clj

```diff
@@ -182,6 +182,59 @@
                               (core/get bulk-ids type))))
              (keys bulk-ids))))
 
+
+(deftest test-bulk-wait_for-test
```

Comment:
I would only use `cli-http-fake` to check if the refresh param is correctly set in the ElasticSearch request